licenses-binary/LICENSE-EPL-1.0.txt

```diff
@@ -0,0 +1,70 @@
+Eclipse Public License - v 1.0
```

Comment:
delete this file, because we don't include EPL 1.0 third-party jars in the binary distribution